### PR TITLE
Fixed every broken formatting in knative serving guide

### PR DESCRIPTION
**Pull request by hamees-sayed**
"Fixes #issue-number" or "Add description of the problem this PR solves"

## Proposed Changes 

-
-
-

### CODE REVIEW

File: docs/serving/using-auto-tls.md [hunk 1]
```diff
@@ -151,8 +151,10 @@ wildcard certificate like `*.my-ns.example.com`.
 
 ### Provising certificates per namespace (wildcard certificates)
 
-!!! warning Provisioning a certificate per namespace only works with DNS-01
- challenge. This component cannot be used with HTTP-01 challenge.
+!!! warning
+
+  Provisioning a certificate per namespace only works with DNS-01
+  challenge. This component cannot be used with HTTP-01 challenge.
```
Comment (snneji) [inline docs/serving/using-auto-tls.md:157]:
```suggestion
!!! warning
    Provisioning a certificate per namespace only works with DNS-01
    challenge. This component cannot be used with HTTP-01 challenge.
```

**Review by snneji — CHANGES_REQUESTED** [review]
Just one small thing otherwise LGTM.

**Comment by hamees-sayed** [comment]
@snneji done

**Comment by snneji** [comment]
Thanks @hamees-sayed!

/lgtm
/approve

**Review by snneji — APPROVED** [review]

**Pull request merged**